electron/actions/app/setAutoupdateSetting.js

```diff
@@ -9,29 +9,38 @@ import setElectronStoreData from '../electronStore/setData.js'
 export default function () {
   let isAutoupdateAvailable
 
-  let isWithAutoupdate =
+  let isAutoupdateBeta
+
+  let isAutoupdateEnabled =
     getElectronStoreKey(
-      'system.isWithAutoupdate'
+      'updates.isAutoupdateEnabled'
     )
 
   if (isWindows) {
     isAutoupdateAvailable = true
 
-    isWithAutoupdate ??= true
+    isAutoupdateBeta = false
```

Comment:
This must be true. Autoupdates don't have the finishing to them yet. 

Main reason being that there's no way for the user to tell if it's downloading the app or not. 

And there's no choice for choosing to autoupdate or declining the update and staying on the initial version.

Reply:
Resolving because code has been updated. Thanks @staniel359!